packages/ui/src/components/va-radio/VaRadio.vue

```diff
@@ -104,6 +109,8 @@ export default defineComponent({
       computedLabel,
       onClick,
       onFocus,
+      computedName: computed(() => props.name || generateUniqueId()),
+      computedTabIndex: computed(() => props.readonly || props.disabled ? -1 : props.tabindex),
```

Comment:
We can move away declaration from `return` to `setup` body.

Reply:
Maxim suggested to move not-reusable final declarations to return.

Reply:
And what if it will become reusable in the future?
Not very convenient to move it back and forth.

Reply:
It makes sense. Think should discuss this pattern at next grooming.